### PR TITLE
updated planet and sea delete-routes

### PR DESCRIPTION
**Pull request by vikiorf**

### CODE REVIEW

File: backend/routes/planet.routes.js [hunk 1]
```diff
@@ -170,13 +170,13 @@ router.delete('/', async (req, res) => {
...
                 success: false,
-                error: `Planet with id ${id} not found`,
+                error: `Planet with id ${id} could not be found`,
             });
         }
 
         res.status(200).json({
             success: true,
-            data: planet,
+            error: 'Planet deleted',
```
Comment (Markussim) [inline backend/routes/planet.routes.js:179]: Error should be empty on status 200.

Here is an example from a SQL delete route (Data is the number of rows affected in the deletion):
```
{
	"success": true,
	"error": "",
	"data": 1
}
```

File: backend/routes/sea.routes.js [hunk 2]
```diff
@@ -94,18 +94,18 @@ router.delete('/', async (req, res) => {
...
     try {
         const deletedSea = await SeaModel.findByIdAndDelete(id);
 
         if (!deletedSea) {
-            return res.status(404).json({ success: false, error: `Sea with id ${id} not found` });
+            return res.status(404).json({ success: false, error: `Sea with id ${id} could not be found` });
         }
 
-        res.status(200).json({ success: true, error: '' });
+        res.status(200).json({ success: true, error: 'Sea deleted' });
```
Comment (Markussim) [inline backend/routes/sea.routes.js:108]: Same as in planet

**Review by Markussim — CHANGES_REQUESTED** [review]
See comments

**Comment by vikiorf** [comment]
Since `model.findByIdAndDelete()` returns the deleted object, I put a static 1 in the response data.

**Pull request merged**